CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # CHANGELOG
 
-## Next
+## 5.2.0
 
-* Your contribution here.
```

Comment:
Please put back the stuff in red, like next version and your contribution here text.